### PR TITLE
Redraw screen after cursor position change.

### PR DESCRIPTION
**Pull request by mjsir911**
See issue #10

### CODE REVIEW

File: autoload/skeleton.vim [hunk 1]
```diff
@@ -133,6 +133,9 @@ function! skeleton#DoDefaultReplacements(filename)
   endif
 
   call skeleton#Replace('CURSOR', '')
+  " Substitutions move the cursor's location, so calling this last sets the
+  " cursor's positions to wherever @CURSOR@ is " Equivalent to :cursor()
```
Comment (noahfrederick) [inline autoload/skeleton.vim]: Looks like there's a typo or two at the end of the comment. How about this instead:

> Move the cursor to the location of the `@CURSOR@` placeholder.

Also please place the comment above the code it describes.

File: autoload/skeleton.vim [hunk 2]
```diff
@@ -133,6 +133,9 @@ function! skeleton#DoDefaultReplacements(filename)
...
+  redraw
```
Comment (noahfrederick) [inline autoload/skeleton.vim]: I'm not sure this belongs here. There's nothing special about the `CURSOR` substitution except that it's the last substitution performed. How about moving this to the end of `skeleton#Load()` before the `return`?

**Comment by mjsir911** [comment]
On reviewing this PR, it seems to be causing more harm than good. 

Using your base branch, there seem to be no issues anymore, and this PR causes some unintended bugs.

**Comment by noahfrederick** [comment]
Interesting. Have you tried a `:redraw!` instead?

**Pull request closed without merging**